### ISSUE TITLE
How can I separate the audio tracks and also change the container back to mp4?

### ISSUE THREAD

**Issue by Savasdotexe** [rdp/screen-capture-recorder-to-video-windows-free#164]
I've tried searching ffmpeg documentation to separate the audio tracks and I see some have asked this already [#119](https://github.com/rdp/screen-capture-recorder-to-video-windows-free/issues/119), [#141](https://github.com/rdp/screen-capture-recorder-to-video-windows-free/issues/141) and it's been suggested, but I'm not sure how to execute custom ffmpeg commands?

I think it's worth having the option to do this under the audio would be useful.

Also importing mkv in Sony Vegas Pro seems to not include the audio so I actually preferred mp4, is there anyway to change this or having this option in the settings would be useful.

**Comment by rdp** [comment]
So you want to leave them as separate instead of merge them, as it currently does? Anyway see 0.13.3beta1 for an mp4 option

**Comment by Savasdotexe** [comment]
> So you want to leave them as separate instead of merge them, as it currently does? Anyway see 0.13.3beta1 for an mp4 option

That's right, I personally prefer to have separate channels, but in some cases when you don't want to mess around with editing the video and want to hear both audio playing when you play the video, it makes sense for them to be merged, so in my opinion a hard coded option to choose would be great.
